moban/definitions.py

```diff
@@ -0,0 +1,21 @@
+class GitRequire(object):
```

Comment:
I expect other data structure would come here, i.e. target, at the moment, is a dictionary but will be evolved to be a class.